src/App.css

```diff
@@ -717,7 +706,86 @@ h1 .num-contexts {
   margin-left: -17px;
   top: -0.15em;
 }
-*/
+
+/* Android Section */
+
+.android .child > .bullet {
+  left: -2px;
+}
+
+.android .child > .bullet::before {
+  content: '▸'; /* •◦◂◄◀︎ ➤▹▸►◥ */
+  position: relative;
+  margin-left: -16.8px;
+  opacity: 0.8;
+  transition: opacity 0.75s ease-in-out;
+}
+
+.android .child > .bullet::after {
+  top: -1px;
+}
+ 
+.android .child.expanded:not(.leaf) > .bullet,
+.android .child.editing:not(.leaf) > .bullet,
+.android .child.cursor-parent:not(.leaf) > .bullet,
+.android .child.cursor-grandparent:not(.leaf) > .bullet {
+  display: inline-block;
+  margin-right: -7px; /* compensate for inline-block */
+  transform: rotate(90deg);
+  transform-origin: -11px 9px;
+  margin-left: -2.8px;
+  margin-top: 1.5px;
+}
+
+.android .dragging {
+  color: green;
+}
+
+.android .child.child.child.dragging > .thought::before {
+  color: green;
+  opacity: 1;
+}
+
+/* Opera Section */
+
+.opera .child > .bullet {
+  left: -2px;
+}
+
+.opera .child > .bullet::before {
+  content: '▸'; /* •◦◂◄◀︎ ➤▹▸►◥ */
+  position: relative;
+  margin-left: -15.8px;
+  opacity: 0.8;
+  transition: opacity 0.75s ease-in-out;
+}
+  
+.opera .child.leaf > .bullet::before {
+  content: '•';
```

Comment:
`content` is no longer needed since it is rendered in the `Bullet` component now.

Reply:
You marked this as resolved but I do not see any change. Am I missing something? Thanks for clarifying!